### PR TITLE
A better HRM 

### PR DESCRIPTION
**Pull request by theKashey**
1. Auto require updated module in `hot`, as long `Electron` does not do it by itself.
- has no effect for webpack (already required)
- does not work for parcel (no module.id)

2. Add notes how to setup HRM.
 - do not use `hot` for parcel bundler. (it does not work, and our example also does not work)
 - how to setup electron
 - be just accurate :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ npm install react-hot-loader@next
 }
 ```
 
+> Note: use `.compilerc` in case of Electron
```
Comment (neoziro) [inline README.md]: I think we should not add note in this section.

File: README.md [hunk 2]
```diff
@@ -47,6 +49,8 @@ const App = () => <div>Hello World!</div>
 export default hot(module)(App)
 ```
 
+Do not use `hot` if you are using **parcel** bundler. It was designed for webpack.
```
Comment (neoziro) [inline README.md]: We should make it work on Parcel too.

Comment (neoziro) [reply README.md]: Can you explain what is the problem? I think we should open a new issue on Parcel, if it does not work with this project, it is useless.

Comment (theKashey) [reply README.md]: https://github.com/parcel-bundler/parcel/issues/619
Meanwhile, I've found a bug in `hot`. From now `hot` will throw an error is `.id` is not defined along the module.

Comment (theKashey) [reply README.md]: Follow up - https://github.com/gaearon/react-hot-loader/pull/818

**Pull request merged**